### ISSUE TITLE
Add support for 'timestamp with time zone' in Accumulo

### ISSUE THREAD

**Issue by RonC-LL** [trinodb/trino#3558]
Trying to do a `CTAS` from a Postgres table that has a field of type `timestamp with time zone` to an Accumulo table threw the following.  The connector author @adamjshook confirms `timestamp with time zone` currently is not supported.

```
Query 20200424_000300_00009_ir7gy failed: Unsupported PrestoType timestamp with time zone
io.prestosql.spi.PrestoException: Unsupported PrestoType timestamp with time zone
	at io.prestosql.plugin.accumulo.model.Field.cleanObject(Field.java:544)
	at io.prestosql.plugin.accumulo.model.Field.<init>(Field.java:66)
	at io.prestosql.plugin.accumulo.model.Field.<init>(Field.java:61)
	at io.prestosql.plugin.accumulo.model.Row.addField(Row.java:82)
	at io.prestosql.plugin.accumulo.io.AccumuloPageSink.appendPage(AccumuloPageSink.java:241)
	at io.prestosql.operator.TableWriterOperator.addInput(TableWriterOperator.java:238)
	[...]
```

**Comment by RonC-LL** [comment]
Quick question so I'm clear: Is this being implemented only *after* pull request #3559 is implemented?  Thanks.

**Comment by adamjshook** [comment]
@RonC-LL Yes, #3559 doesn't address this issue.  It still needs support.  I'm hoping to take a look at it soon.
